README.md

```diff
@@ -37,10 +38,69 @@ To publish this example to your own Pactflow account and view it in the UI follo
+
+2. Next we can use Schemathesis to test the API against our generated swagger doc. Start the example project in Visual Studio or via dotnet cli from the project root:
+
+    ```
+    dotnet run --project .\example-bi-directional-provider-dotnet
+    ```
+    In powershell in the root directory (you'll need a new window if you are using dotnet CLI to run the app) use docker run to perform the Schemathesis test. This will check the API implements the swagger doc accurately and output the results to report.txt. Once this is generated the API can be shut down again
+
+    ```
+    docker run --net="host" schemathesis/schemathesis:stable run --stateful=links --checks all http://host.docker.internal:9000/swagger/v1/swagger.json > report.txt
+    ```
+
+3. Now that the Swagger doc is generated and verified the contract can be published to Pactflow. The easiest way to do this via windows is using postman to make an HTTP PUT request to the Pactflow provider contract API endpoint. First base64 encode the report and swagger docs. This can be done in powershell using the following commands:
+    ```
+    [convert]::ToBase64String((Get-Content -path "src/example-bi-directional-provider-dotnet/swagger.json" -Encoding byte))
+    ```
+
+    ```
+    [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes((Get-Content -path "report.txt")))  
+    ```
+    Open postman and create a new PUT request to the provider contracts endpoint, the git_commit section will be used as the version for the provider contract.
+
+    `https://<your_pactflow_url/contracts/provider/<providerName>/version/<git_commit>`
+
+    In the Authorization section select Bearer Token and add the read/write token from your Pactflow account.
+    In the headers tab also set the "content-type" header to "application/json".
+    In the body tab select "raw" and "JSON" as the content type.
+
+    The format for the body is as follows, paster in your encoded report and swagger doc content as shown.
+
+    ```
+    {
+        "content":"<swagger doc encoded content here>",
+        "contractType": "oas",
+        "contentType": "application/yaml",
+        "verificationResults": {
+            "success": true,
+            "content": "<report.txt encoded content here>",
+            "contentType": "text/plain"	,
+            "verifier": "Schematheis"
+        }
+
+    }
+    ```
+    Send the request and when you have a 200 response the provider contract will be visible in your Pactflow account.
+
+ 4. Check can-i-deploy to see if your provider contract is compatible with current consumers. In Powershell set the environment variable $env:PACTICIPANT to the name of the provider in the url for step 3. Set the environment variable $env:GIT_COMMIT to the same thing as in step 3. This tells pactflow which provider contract and version to perform the can-i-deploy check on. Run the can-i-deploy check using the pact-cli docker container:
```

Comment:
Check `can-i-deploy` to see if your provider contract is compatible with current consumers. 

In Powershell 
- set the environment variable `$env:PACTICIPANT` to the name of the provider in the url for step 3. 
- set the environment variable `$env:GIT_COMMIT` to the same as in step 3. 

This tells pactflow which provider contract and version to perform the `can-i-deploy` check on. 

Run the `can-i-deploy` check using the pact-cli docker container: